src/Esfa.Vacancy.Register.Api/Controllers/GetApprenticeshipVacancyController.cs

```diff
@@ -55,5 +57,14 @@ public GetApprenticeshipVacancyController(GetApprenticeshipVacancyOrchestrator a
 
             return Ok(vacancy);
         }
+
+        [HttpGet]
+        [AllowAnonymous]
+        [Route("{wrongRoute}", Order = 1)]
+        [ApiExplorerSettings(IgnoreApi = true)]
+        public HttpResponseMessage Get(string wrongRoute)
+        {
+            return Request.CreateErrorResponse(HttpStatusCode.NotFound, "You have not searched by vacancy reference number.");
```

Comment:
When we do stuff like this do we not want a Vacancy API error code? Also could we not throw something here that is then handled by the ExceptionHandler thingy?

Reply:
We have already agreed to accept string for these end points, perhaps make that change now, instead having two endpoints.

Reply:
I understand why you had to add an endpoint to to handle an invalid request. Once we change the other Get to accept strings then we can kill this endpoint off. As Deven has mentioned whether we make that change now or later is up for debate :)